orb-slot-ctrl/src/test_utils.rs

```diff
@@ -0,0 +1,78 @@
+use std::fs;
+
+use crate::{
+    efivar::{bootchain::BootChainEfiVars, rootfs::RootfsEfiVars},
+    EfiVarDb, OrbSlotCtrl,
+};
+use tempfile::TempDir;
+
+pub struct Fixture {
+    _tempdir: TempDir,
+    pub db: EfiVarDb,
+    pub slot_ctrl: OrbSlotCtrl,
+    pub bootchain: BootChainEfiVars,
+    pub rootfs: RootfsEfiVars,
+}
+
+impl Default for Fixture {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl Fixture {
+    pub fn new() -> Self {
+        let tempdir = TempDir::new_in("/tmp").unwrap();
+        let db_path = tempdir.path().join("sys/firmware/efi/efivars/");
+        fs::create_dir_all(&db_path).unwrap();
+
+        let db = EfiVarDb::from_rootfs(&tempdir).unwrap();
+        let bootchain = BootChainEfiVars::new(&db).unwrap();
+        let rootfs = RootfsEfiVars::new(&db).unwrap();
+        let slot_ctrl = OrbSlotCtrl::new(&db).unwrap();
+
+        bootchain
+            .current
+            .create_and_write(&[0x07, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00])
```

Comment:
Explain these magic values.

Reply:
I got them all from the unit tests from `rootfs.rs`  and `bootchain.rs`, and commits were done by you and Mark Eder, so you probably can explain them better than me :sweat_smile: